android/Buyo/app/src/main/java/com/cmpe352group4/buyo/ui/login/LoginFragmentVendor.kt

```diff
@@ -68,6 +70,16 @@ class LoginFragmentVendor : BaseFragment() {
         googleMapButtonListener()
     }
 
+    override fun onResume() {
+        super.onResume()
+
+        val lat: String? = sharedPref.getVendorLat()
+        val lon: String? = sharedPref.getVendorLon()
+        if (lat != "" && lon != ""){
```

Comment:
I could not understand the logic here. Why should vendor reselect the location if latitude and longitude are not empty? Doesn't vendor need to reselect location if latitude or longitude is empty?

Reply:
Vendor should choose a location only when signing up. Rechoosing location means, in this context, something else. Basically, when a non-registered vendor goes to sign up page, they will see "Choose Company Location".  When they choose one and return back to the sign up page, this button will be "Rechoose Location", since it was choosen just before. 

Rechoosing location for a registered user will be implemented in the profil page later.

Reply:
Okey, got it 👍